lib/pages/detail/no_score_message.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:pola_flutter/theme/assets.gen.dart';
+import 'package:pola_flutter/theme/colors.dart';
+import 'package:pola_flutter/theme/fonts.gen.dart';
+import 'package:pola_flutter/theme/text_size.dart';
+import 'package:pola_flutter/i18n/strings.g.dart';
+
+class NoScoreMessage extends StatelessWidget {
+  const NoScoreMessage({Key? key, required this.t}) : super(key: key);
+
+  final Translations t;
```

Comment:
Jeśli masz context, to nie musisz przekazywać tłumaczeń. W metodzien build, linijkę przed returnem możesz dać `final Translations t = Translations.of(context); `, wtedy możesz linijkę 9 i 11 usunąć.